packages/repository/test/acceptance/relations/hasMany/repositories/customer.repository.ts

```diff
@@ -12,12 +12,13 @@ import {
   DefaultCrudRepository,
   DataSourceType,
 } from '../../../../../src/repositories';
+import {DefaultHasManyEntityCrudRepository} from '../../../../../src/repositories/relation.repository';
 
-export class CustomerRepository extends DefaultCrudRepository<
-  Customer,
+export class CustomerRepository extends DefaultHasManyEntityCrudRepository<
+  Order,
```

Comment:
I find it suspicious that CustomerRepository is extending HasManyEnityCrudRepository with type Order.

I'd expect that CustomerRepository remains without changes, because we may want to access customer models directly, without going through any order.

Reply:
That is correct! I will revert those changes.